docs/welcome/why-kubeedge.md

```diff
@@ -3,15 +3,15 @@ title: Why KubeEdge
 sidebar_position: 1
 slug: /
 ---
-**KubeEdge** is an open source system extending native containerized application orchestration and device management to hosts at the Edge. It is built upon Kubernetes and provides core infrastructure support for networking, application deployment and metadata synchronization between cloud and edge. It also supports MQTT and allows developers to author custom logic and enable resource constrained device communication at the Edge. KubeEdge consists of a cloud part and an edge part. Both edge and cloud parts are now open-sourced.
+**KubeEdge** is an open-source system that enhances Kubernetes by extending its capabilities to manage and orchestrate containerized applications and devices to host at the Edge. It is built upon Kubernetes and provides core infrastructure support for networking, application deployment, and metadata synchronization between cloud and edge. It also supports MQTT, a lightweight messaging protocol, and allows developers to write custom logic and enable resource-constrained device communication at the Edge. KubeEdge consists of a cloud part and an edge part. Both edge and cloud parts are now open-sourced.
```

Comment:
I think we don't need the `-` for `open source` @Himanshu-kiran

Reply:
May be or may be not. I think we can use - in these cases like open-source in 1st line and if line starting with open source then without -.
If you would say, I would remove - to maintain consistency and remove confusion throughout the doc without using -  .
@fisherxu

Reply:
Let's keep consistency :)

Reply:
@fisherxu  now please check it.